core/src/main/scala/spinal/core/sim/package.scala

```diff
@@ -556,6 +556,21 @@ package object sim {
     def toDouble: Double = this.toBigDecimal.doubleValue
 
   }
+  
+  /**
+    * Add implicit function to BigInt
+    */
+  implicit class SimBigIntPimper(x: BigInt) {
+    def toBytes(bits: Int = -1, byteOrderReverse: Boolean = false): Array[Byte] = {
```

Comment:
Emm, not strictly, but endianness is meaningful enough.

Reply:
> Ahh instead of having byteOrderReverse, we can use :
> 
> https://github.com/SpinalHDL/SpinalHDL/blob/c996c5d7a0e92098e64b9ee3e633b87d163c01f9/lib/src/main/scala/spinal/lib/Misc.scala#L25
> 
> Being by default to LITTLE ?

One problem is that the endianness is defined in the lib package.
I think it's ok to use core.xxxx in the lib, but the other way around is not so good practice.
How about moving endianness to the core package?

Reply:
> Emm, not strictly

Yes i think you are right :)

> One problem is that the endianness is defined in the lib package.
I think it's ok to use core.xxxx in the lib, but the other way around is not so good practice.
How about moving endianness to the core package?

Ok with it, can be moved.